distributed/tests/test_scheduler.py

```diff
@@ -1447,9 +1447,6 @@ async def test_learn_occupancy_2(c, s, a, b):
 async def test_occupancy_cleardown(c, s, a, b):
     s.validate = False
 
-    # Inject excess values in s.occupancy
-    s.workers[a.address].occupancy = 2
-    s.total_occupancy += 2
```

Comment:
We can probably delete this test. It's mutating state and ensures that it is reset again. That's quite unnatural